### DISCUSSION TITLE
Release candidate 3.21.0rc1 Testing

### DISCUSSION THREAD

**Discussion by sotorrio1** [CCSI-Toolset/FOQUS#1213] [Show and tell]
I ran through the testing in here: https://github.com/CCSI-Toolset/devops/blob/main/release_candidate_testing.md

Everything works but there are a couple of items to discuss:

- When I run any type of regular SDoE (without the Dask flag) I can see the following ERROR in the console 
![image](https://github.com/CCSI-Toolset/FOQUS/assets/33531919/239cd33e-5e59-4908-b036-720bdfad62c9)

- There are a few places in the code for UQ, SDoE and ODoE that use the file [foqus_lib/framework/sampleResults/results.py](https://github.com/CCSI-Toolset/FOQUS/blob/master/foqus_lib/framework/sampleResults/results.py) and during all these tutorials I got some FutureWarning messages about an upcoming deprecation within pandas. I've opened issue #1212 to take care of this after the release.

**Comment by sotorrio1** [comment]
Issue #1212  should be fixed with PR #1214

**Comment by lbianchi-lbl** [comment]
I think the Dask error being shown could be fixed by using `logging.info()` instead of `.exception()` here:

https://github.com/CCSI-Toolset/FOQUS/blob/47053af44897bbbe5b0b79c19421f40cd6d9b124/foqus_lib/framework/sdoe/sdoe.py#L96-L104

I think this would be more in line with the semantics of the check (the user should be notified, but nothing would stop working per se) without having to make changes to the code. I'll open a PR with this change (#1215).
